plugins/modules/spectrum_model_attrs.py

```diff
@@ -9,110 +9,108 @@
-         - 'Currently defined names are:'
-         - '                 C(App_Manufacturer) (C(0x230683))'
-         - '                 C(CollectionsModelNameString) (C(0x12adb))'
-         - '                 C(Condition) (C(0x1000a))'
-         - '                 C(Criticality) (C(0x1290c))'
-         - '                 C(DeviceType) (C(0x23000e))'
-         - '                 C(isManaged) (C(0x1295d))'
-         - '                 C(Model_Class) (C(0x11ee8))'
-         - '                 C(Model_Handle) (C(0x129fa))'
-         - '                 C(Model_Name) (C(0x1006e))'
-         - '                 C(Modeltype_Handle) (C(0x10001))'
-         - '                 C(Modeltype_Name) (C(0x10000))'
-         - '                 C(Network_Address) (C(0x12d7f))'
-         - '                 C(Notes) (C(0x11564))'
-         - '                 C(ServiceDesk_Asset_ID) (C(0x12db9))'
-         - '                 C(TopologyModelNameString) (C(0x129e7))'
-         - '                 C(sysDescr) (C(0x10052))'
-         - '                 C(sysName) (C(0x10b5b))'
-         - '                 C(Vendor_Name) (C(0x11570))'
-         - '                 C(Description) (C(0x230017))'
-         - Hex IDs are the direct identifiers in Spectrum and will always work.
-         - 'To lookup hex IDs go to the UI: Locator -> Devices -> By Model Name -> <enter any model> -> Attributes tab.'
-         type: str
-         required: true
-       value:
-         description:
-         - Attribute value. Empty strings should be V("") or V(null).
-         type: str
-         required: true
-'''
-
-EXAMPLES = r'''
+  url:
+    description:
+      - URL of OneClick server.
+    type: str
+    required: true
+  url_username:
+    description:
+      - OneClick username.
+    type: str
+    required: true
+    aliases: [username]
+  url_password:
+    description:
+      - OneClick password.
+    type: str
+    required: true
+    aliases: [password]
+  use_proxy:
+    description:
+      - If V(false), it will not use a proxy, even if one is defined in an environment variable on the target hosts.
+    default: true
+    required: false
+    type: bool
+  name:
+    description:
+      - Model name.
+    type: str
+    required: true
+  type:
+    description:
+      - Model type.
+    type: str
+    required: true
+  validate_certs:
+    description:
+      - Validate SSL certificates. Only change this to V(false) if you can guarantee that you are talking to the correct endpoint and there is
+        no man-in-the-middle attack happening.
+    type: bool
+    default: true
+    required: false
+  attributes:
+    description:
+      - A list of attribute names and values to enforce.
+      - All values and parameters are case sensitive and must be provided as strings only.
+    required: true
+    type: list
+    elements: dict
+    suboptions:
+      name:
+        description:
+          - Attribute name OR hex ID.
+          - Currently defined names are:.
```

Comment:
Maybe you should adjust the code so that a trailing `:` (or `!` or `;`, if that's not already handled) doesn't get another trailing period.
```suggestion
          - Currently defined names are:
```

Reply:
Yup, I have updated the code to `    if line.endswith((".", "!", ":", ";", ",")):`. I thought of adding `?` as well, but I thought it would be weird to see a question in a documentation. I left that bridge to be crossed when we get there.

Reply:
I'd still also add `?`, since adding a period after a question mark is even more weird ;-)